app/helpers/hosts_global_status_helper.rb

```diff
@@ -0,0 +1,46 @@
+module HostsGlobalStatusHelper
```

Comment:
Rubocop didn't like that i extended `hosts_helper` to more lines, so I extracted this one out